build/yamls/antrea-migrator.yml

```diff
@@ -0,0 +1,59 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: antrea
+    component: antrea-migrator
+  name: antrea-migrator
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-migrator
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-migrator
+    spec:
+      hostPID: true
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: linux
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      serviceAccountName: antrea-agent
+      volumes:
+        - name: cni-cfg
+          hostPath:
+            path: /etc/cni/net.d
+        - name: containerd
+          hostPath:
+            path: /run/containerd
+      initContainers:
+        # initContainer will kill all sandboxes container
+        - name: antrea-migrator-init
+          image: antrea/antrea-migrator:latest
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          command:
+            - restart_sandbox
+          volumeMounts:
+            - mountPath: /run/containerd
+              name: containerd
+            - mountPath: /etc/cni/net.d
+              name: cni-cfg
+      containers:
+        - image: busybox:1.36
```

Comment:
Why is this needed? If you want this container to be a long run process, can the image `antrea/antrea-migrator` be reused?